File: app/src/main/java/com/example/waves/zamza/DebtRepaymentTermFragment.java
package com.example.waves.zamza;

import android.os.Bundle;
import android.support.v4.app.Fragment;
import android.view.LayoutInflater;
import android.view.View;
import android.view.ViewGroup;

public class DebtRepaymentTermFragment extends Fragment {
    @Override
    public View onCreateView(LayoutInflater inflater , ViewGroup container , Bundle savedInstanceState){
        View view = inflater.inflate(R.layout.fragment_debt_repayment_term , container , false);
        return view;
    }
}

File: app/src/main/java/com/example/waves/zamza/DebtRepaymentTermFragment.kt
package com.example.waves.zamza

import android.os.Bundle
import android.support.v4.app.Fragment
import android.view.LayoutInflater
import android.view.View
import android.view.ViewGroup

class DebtRepaymentTermFragment : Fragment() {
    override fun onCreateView(inflater: LayoutInflater, container: ViewGroup?, savedInstanceState: Bundle?): View? {
        return inflater.inflate(R.layout.fragment_debt_repayment_term, container, false)
    }
}